.github/workflows/release.yml

```
@@ -19,25 +19,25 @@ jobs:
+        GOOS=linux GOARCH=arm64 go build -o ccstat-linux-arm64 ./cmd/ccstat
         
         # macOS amd64
-        GOOS=darwin GOARCH=amd64 go build -o ccmonitor-darwin-amd64 ./cmd/ccmonitor
+        GOOS=darwin GOARCH=amd64 go build -o ccstat-darwin-amd64 ./cmd/ccstat
         
         # macOS arm64 (Apple Silicon)
-        GOOS=darwin GOARCH=arm64 go build -o ccmonitor-darwin-arm64 ./cmd/ccmonitor
+        GOOS=darwin GOARCH=arm64 go build -o ccstat-darwin-arm64 ./cmd/ccstat
     
     - name: Create Release
       uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)